GETTING_STARTED.md

```diff
@@ -0,0 +1,106 @@
+# Getting Started With Convox Praxis
+
+Convox Praxis is a universal infrastructure interface. When you develop and deploy applications using Praxis you completely abstract away concerns about where your application is running. In minutes you can set up a system that has perfect development / production parity and deploy your app to the cloud.
+
+This guide will walk you through installing the Praxis CLI, the Rack deployment platform, and deploying an application.
```

Comment:
This guide will walk you through installing the Praxis CLI and developing and deploying an application.